docs/contributor-guide/developer-guide/technical-contribution-guide.md

```diff
@@ -18,9 +18,58 @@ Development tasks are managed in the [GitHub issues](https://github.com/civiform
 
 If you're just getting started, check out issues labeled with [Good First Issue](https://github.com/civiform/civiform/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). Also check out issues in the [next milestone](https://github.com/civiform/civiform/milestones?direction=asc\&sort=due\_date\&state=open) so you can work on the highest-priority tasks.
 
+### Git workflow
+
+Get the latest state from the github repository and create a new local development branch:
+
+```sh
+git fetch origin
+git checkout -b your_username/dev-branch-description
+git reset origin/main
```

Comment:
Why suggest a git reset here?  It's a dangerous command that I wouldn't want people getting muscle memory for.  Especially since you wouldn't be using it if the branch already exists.

A better pattern would be:
- git checkout main
- git pull
- git checkout -b your_username/dev-branch-description

Reply:
Removed the use of reset.